### PR TITLE
learning budget overuse  and restrictions

### PR DESCRIPTION
**Pull request by peterlangmar**
Cassus wants to spend over 2000 EUR this year, which is a big investment for LabCoop. We need to have a system, which enable us to make such investment, but also allows us to only make investments, which have a potential return, otherwise it is not safe to fail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -446,13 +446,16 @@ In case of Holidays and Home office, letting them know within this period doesn'
 - All costs must be within a common sense limit. No ***** hotel, no luxury.
 
 ##### Policy: Company financed learning (for everyone and all circles in the Lab)
-- Up to 2000 EUR + VAT / person / year (incl. travel costs)
-- Monthly max. 3000 EUR + VAT on the company level. Biz role approval is needed about this monthly budget.
+- Up to 2000 EUR + VAT / person / year (incl. travel costs), 166.7 EUR / month in average
+- Monthly max. 3000 EUR + VAT in total on the company level. Biz role approval is needed about this monthly budget.
```
Comment (cassus) [inline README.md]: How to track, manage the company level limit?

**Comment by cassus** [comment]
:exclamation: Need a start date so it's not unilaterally and retroactively overriding previous agreements between Lab.Coop and Employees (me).

File: README.md (hunk 1 above)
Comment (peterlangmar) [inline README.md]: This is not a change in the policy. The google sheet could have a monthly sum cell.

**Comment by peterlangmar** [comment]
@cassus I would like to discuss this with you in person to better understand your concern as you are the only one, who would be impacted by this policy section.

**Comment by peterlangmar** [comment]
@cassus I've amended your idea, which you have shared on Slack with me

File: README.md [hunk 2]
```diff
@@ -446,13 +446,17 @@ In case of Holidays and Home office, letting them know within this period doesn'
...
+- Learning budget over 2000 EUR / year is the domain of the Biz role as it becomes an investment for the company. If the Partner leaves the company before his/her cummulative learning budget balance is over the 2000 EUR / year average, the person need to pay it back proportionally to the company. (For example the Partner has worked 24 months for the company and has spent 6000 EUR on learning, then the Partner owes 1000 EUR to the company.) This policy is in force from the 23rd of November 2015. (Before that date there was only one relevant case: Adam Banko agreed to stay at least one more year after participating at the Practitioner Certification Training (Vienna), 13th - November 17th, 2015.) 
+- In case a Partner does not want to use a larger learning budget, as the agreed one (just as described above), s/he has the option to access the budget against his or her preferred dividend stocks. (For example 10k HUF budget overuse would mean getting 10k less preferred dividend stocks in that given year.) In this case there is no employment commitment or repayment obligation.
```
Comment (peterlangmar) [inline README.md]: @cassus your idea in words

File: README.md [hunk 3]
```diff
@@ -446,13 +446,17 @@ In case of Holidays and Home office, letting them know within this period doesn'
...
+- If a Partner spends on learning during his/her probation time and decides to leave before the probation ends s/he has to pay back the amount s/he spent on learning.
+- If a Partner leaves the company, s/he needs to pay the balanec over the proportional usage of the learning budget in that given year. (For example if the Partner leaves by the end of Oct and her/his learning budget for the given year was 1800 EUR than s/he owes the company 133 EUR (2000/12*10-
+- Learning budget over 2000 EUR / year is the domain of the Biz role as it becomes an investment for the company. If the Partner leaves the company before his/her cummulative learning budget balance is over the 2000 EUR / year average, the person need to pay it back proportionally to the company. (For example the Partner has worked 24 months for the company and has spent 6000 EUR on learning, then the Partner owes 1000 EUR to the company.) This policy is in force from the 23rd of November 2015. (Before that date there was only one relevant case: Adam Banko agreed to stay at least one more year after participating at the Practitioner Certification Training (Vienna), 13th - November 17th, 2015.)
```
Comment (peterlangmar) [inline README.md]: @cassus your request of starting date with stating your special, pre-policy agreement with the biz role

**Comment by cassus** [comment]
:+1: I'd choose the _preferred dividend stock_ option.

**Comment by peterlangmar** [comment]
@cassus for the first holacracy training that would also mean using a policy backwards. Why would it be ok for you in this case?

**Comment by cassus** [comment]
I mean for the Amsterdam training.

**Comment by peterlangmar** [comment]
I think we should be more flexible

**Comment by cassus** [comment]
:white_check_mark:

**Pull request merged**